source/generate_uids/generate_storage_sopclass_uids.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+"""
+Create UIDs for the Storage SOP Classes taken from the generated ``_uids.py``.
+"""
+import os
+import re
+import tokenize
+
+import sys
+
+import pydicom
+
+from pydicom._uid_dict import UID_dictionary
+
+STORAGE_REGEX = re.compile('.*(Storage|Storage SOP Class|Storage - '
+                           'For Presentation|Storage - For Processing)$')
+
+IS_PYTHON3 = sys.version_info >= (3,)
```

Comment:
`pydicom.compat` has an `in_py2`.  Would prefer to use that, as I could see dropping all python 2 in a year or two and it would be easier if same variable used everywhere.